connectors/sources/outlook.py

```diff
@@ -456,6 +485,78 @@ async def get_user_accounts(self):
                 yield user_account
 
 
+class MultiOffice365Users(BaseOffice365User):
+    """Fetch multiple Office365 users based on a list of email addresses."""
+
+    def __init__(self, client_id, client_secret, tenant_id, client_emails: List[str]):
+        super().__init__(client_id, client_secret, tenant_id)
+        self.client_emails = client_emails
+
+    async def get_users(self):
+        access_token = await self._fetch_token()
+        errors = []
+        for i in range(0, len(self.client_emails), 20):
```

Comment:
Would be great to make it a constant at least (20). Also why was 20 chosen as batch size?

Reply:
I've updated the code to use the `GRAPH_API_BATCH_SIZE` constant instead of hardcoding the value `20`, see commit 
e4923b5.

The batch size of `20` is due to the max limit set by the Graph API. For ref you can check the [Graph API documentation on JSON batching](https://learn.microsoft.com/en-us/graph/json-batching#:~:text=JSON%20batch%20requests%20are%20currently%20limited%20to%2020%20individual%20requests)